server/issue.go

```diff
@@ -672,7 +671,7 @@ func (p *Plugin) AttachCommentToIssue(in *InAttachCommentToIssue) (*jira.Comment
 		conf := instance.Common().getConfig()
 		extraText := ""
 		for _, fileID := range post.FileIds {
-			mattermostName, jiraName, mime, e := client.AddAttachment(p.API, in.IssueKey, fileID, conf.maxAttachmentSize)
+			mattermostName, jiraName, mime, e := client.AddAttachment(*p.client, in.IssueKey, fileID, conf.maxAttachmentSize)
```

Comment:
Go lint error `server/issue.go:674:36: shadow: declaration of "err" shadows declaration at line 633 (govet)` is thrown in this case, since `err` is already declared at line 633. I will have to let it be as is.